pages/contribute.md

```diff
@@ -3,24 +3,119 @@ layout: default
 title: Contribute to Auto-WCAG
 ---
 
-Auto-WCAG is a group open to anyone interested in accessibility, accessibility testing or automated testing. We welcome contributions from anyone, either though Github issues and pull requests, or by contribution to our monthly teleconferences.
+Auto-WCAG is a [W3C](https://www.w3.org/) [Community Group](https://www.w3.org/community/) open to anyone interested in accessibility or accessibility testing, manually or automated. 
 
-## Leave your feedback on GitHub
+We welcome contributions from anyone, and there are many ways to contribute, from Github issues and pull requests, to contribution to our monthly teleconferences.
 
-Contributing is open to anyone. We welcome any pull requests for changes. We ask that you make changes to pull requests **only for drafts**. Before contributing new rules, we recommend you checked their validity with several experienced accessibility auditors first. This helps you identify potential stumbling blocks early in their design.
+You don't have to be a member of the Auto-WCAG Community Group to contribute by [giving feedback on rules](#give-feedback-on-rules) or [suggesting new rules](#suggest-new-rules).
```

Comment:
`You don't have to be a member of the Auto-WCAG Community Group to contribute.` should suffice above. Because below you are going to greater detail of how to contribute, so no need to expand on them here in my opinion.

Reply:
Fixed